daml-script/runner/BUILD.bazel

```diff
@@ -63,14 +63,16 @@ da_scala_library(
 da_scala_test(
     name = "tests",
     srcs = glob(
-        ["src/test/**/*.scala"],
+        ["src/test/scala/**/*.scala"],
```

Comment:
Changes in this file support the use of a `resource` and `scala` directory in `daml-script/src/test`